contracts/Strategy.sol

```diff
@@ -484,15 +481,15 @@ contract Strategy is BaseStrategy {
         }
 
         reservedAmount = borrowBalance.mul(1e18).div(tempColla);
-
         if(depositBalance >= reservedAmount){
             uint256 redeemable = depositBalance.sub(reservedAmount);
-
-            if (redeemable < _amount) {
-                cToken.redeemUnderlying(redeemable);
-
-            } else {
-                cToken.redeemUnderlying(_amount);
+            uint256 balan = cToken.balanceOf(address(this));
```

Comment:
important to recheck 

did this to avoid rounding errors

Reply:
```suggestion
            uint256 balance = cToken.balanceOf(address(this));
```